tests/open_alchemy/helpers/test_peek/test_peek_extension.py

```diff
@@ -119,8 +149,19 @@ def test_primary_key_wrong_type():
         pytest.param({}, None, id="missing"),
         pytest.param({"x-primary-key": False}, False, id="False"),
         pytest.param({"x-primary-key": True}, True, id="True"),
+        pytest.param(
+            {"x-open-alchemyprimary-key": False},
```

Comment:
is the problem here that it should be `x-open-alchemy-primary-key`? Note the `-` between `x-open-alchemy` and `primary-key`.

Reply:
Good catch! that's probably the problem then. I will remove the `xfail` marker.